src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -0,0 +1,99 @@
+@import "scss/constants.scss";
+
+$grey--regular: #3f3d42;
+$grey--hover: #4c494f;
+$grey--users: #4c494f;
+$grey--ligth: #ebebeb;
+
+$purple--regular: #7c46fb;
+$purple--hover: #7c46fb;
+
+$ScheduleEvent--margin-top: 0.25rem;
+$ScheduleEvent--height: 3.75rem;
+$ScheduleEvent--padding: 0 0.25rem 0 0.75rem;
+$ScheduleEvent--border-radius: 18px;
+$ScheduleEvent--box-shadow: 0 5px 10px rgba(0, 0, 0, 0.65);
+
+$bookmark--padding: 0.5rem;
+$bookmark-hover--padding-top: 0.375rem;
```

Comment:
[NB] Any reasons to have them in rem?

Reply:
Ref:

- https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#relative_length_units
  - > Relative length units are relative to something else, perhaps the size of the parent element's font, or the size of the viewport. The benefit of using relative units is that with some careful planning you can make it so the size of text or other element scales relative to everything else on the page.
  - > `rem`: Font size of the root element.